### PR TITLE
[PLAYER-3700] Added UI for playback speed feature

### PR DESCRIPTION
**Pull request by MarkEPAM**
- added playback speed panel
- added playback speed button 
- added bridge methods for handling playback speed events
Unit tests not included.
Task: https://jira.corp.ooyala.com/browse/PLAYER-3700

### CODE REVIEW

File: sdk/react/controlBar.js [hunk 1]
```diff
@@ -33,9 +33,12 @@ var ControlBarWidget = require('./widgets/controlBarWidgets');
 var CollapsingBarUtils = require('./collapsingBarUtils');
 var VolumeView = require('./widgets/VolumeView');
 var ResponsiveDesignManager = require('./responsiveDesignManager');
-
 var styles = Utils.getStyles(require('./style/controlBarStyles.json'));
 
+const constants = {
+  playbackSpeedRatePostfix: "x"
```
Comment (pilievOoyala) [inline sdk/react/controlBar.js:39]: This is probably fine but do some languages use a different symbol for multiplier? Should this be overridable?

**Comment by pilievOoyala** [comment]
LGTM.

File: sdk/react/controlBar.js (hunk 1 above)
Comment (pilievOoyala) [reply sdk/react/controlBar.js:39]: Disregard this. I asked JP and we don't allow you to customize it on web so you don't need to either.

**Review by brol1dev — APPROVED** [review]
LGTM

**Pull request merged**